docs/UserGuide.md

```diff
@@ -163,35 +233,105 @@ Mark the specified task from the planner as done.
 
 
 ### Edit task: `edit` or `e`
-Edits the specified task from the planner.
+This command edits a specific task in the planner.
 
-#### Format
-`edit INDEX [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
-`e INDEX [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
-* At least one parameter have to be edited.
+The guide for this command is split into three sections:
 
-#### Parameters
-* `INDEX` refers to the index number shown in the displayed task list. It cannot be a negative integer.
+* [Basic Usage](#basic-usage) (For users new to command line applications)
+* [Intermediate Usage](#intermediate-usage) (For users comfortable with command line applications)
+* [Advanced Usage](#advanced-usage) (Extension to *Intermediate Usage*)
+
+#### Basic Usage
+
+The `edit` command accepts 8 parameters.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to edit in the displayed task list. It must be a non-negative integer.
+
+At least one parameter shown below have to be edited:
 * `NAME` is the name of the task.
+
 * `TIME` is the time to start doing this task (**HHMM**). For example, 1100.
+
 * `DURATION` is the expected time taken to complete task (in hours).
+
 * `DEADLINE` is the date to do task by (**DD-MM-YYYY**). For example, 16-09-2020.
+
 * `RECURRENCE` is one of **TODAY**, **MONDAY**, **TUESDAY**, **WEDNESDAY**, **THURSDAY**, **FRIDAY**, **SATURDAY**, **SUNDAY** or a specific date (**DD-MM-YYYY**).
+
 * `IMPORTANCE` is one of **HIGH**, **MEDIUM**, **LOW**.
+
 * `ADDITIONAL-NOTES` is the additional notes of the task.
-   
-#### Example usage
 
-Use [list feature](#list-tasks-list-or-l) first to get the `INDEX` of the task to be edited.
-* `edit 1 n/Assignment2 t/1800 D/17-09-2020 d/1 r/today i/high a/Refer to lecture notes`
-* `edit 1 t/1800`
-* `e 1 n/Assignment2 t/1800 D/17-09-2020 d/1 r/today i/high a/Refer to lecture notes`
+##### Command Format
 
-#### Expected output
+`edit TASK-ID [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+
+##### Example Usage
+
+Firstly, use [list feature](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be edited.
```

Comment:
`use [list feature](#list-tasks-list-or-l)`

Would calling it list command be better than feature?